pkgs/data/themes/catppuccin-bat/default.nix

```diff
@@ -0,0 +1,40 @@
+{ fetchFromGitHub
+, lib
+, stdenvNoCC
+, variant ? "macchiato"
+}:
+let
+  pname = "catppuccin-bat";
+  validVariants = [ "latte" "frappe" "macchiato" "mocha" ];
+in
+lib.checkListOfEnum "${pname}: color variant" validVariants [ variant ]
+
+  stdenvNoCC.mkDerivation
+{
```

Comment:
```suggestion
stdenvNoCC.mkDerivation {
```

Reply:
Sorry, my nixpkgs-fmt kept making that change on me... i thought it preferred it this way too before.